test/async-hooks/test-graph.signal.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
```

Comment:
yeah, I get the purpose of the test, just not why it doesn't follow the naming pattern of other tests (specifically, why `test-graph.signal` instead of `test-graph-signal`)